### PR TITLE
Add dm_control example for SAC

### PR DESCRIPTION
**Pull request by frederikschubert**
This PR introduces the common squashed normal distribution for the SAC policy on `dm_control` and provides an example that solves the `walker.walk` task. Interestingly clipping the actions to the range `[-1, 1]` diverges.
rendering

@KristianHolsheimer How would you go about changing the installation script for this notebook to add `dm_control` as a dependency?

### CODE REVIEW

**Comment by KristianHolsheimer** [comment]
Perhaps it would be good to add `install_extras={'box2d': [...], 'atari': [...], 'dm_control': [...]}` in `setup.py` to avoid depending on all those envs by default. You would then have to install using e.g.
```
$ pip install coax[dm_control]
```
But for now I would just add it to `requirements.txt`. If it becomes an issue we can always separate them out later.

**Comment by frederikschubert** [comment]
Sounds good  👍

**Comment by KristianHolsheimer** [comment]
Awesome, thanks!

File: coax/proba_dists/_normal.py [hunk 1]
```diff
@@ -81,17 +100,24 @@ def check_shape(x, name, flatten):
...
+                x = jnp.clip(x, *self.clip_logvar)
             return x
 
         def sample(dist_params, rng):
             mu = check_shape(dist_params['mu'], name='mu', flatten=True)
             logvar = check_shape(dist_params['logvar'], name='logvar', flatten=True)
 
             X = mu + jnp.exp(logvar / 2) * jax.random.normal(rng, mu.shape)
+            if self.squash:
+                X = jnp.tanh(X) * self._scale + self._offset
```
Comment (KristianHolsheimer) [inline coax/proba_dists/_normal.py:113]: I think used an env wrapper for this in the past. Have you seen [coax.wrappers.BoxActionsToReals](https://coax.readthedocs.io/en/latest/coax/wrappers.html#coax.wrappers.BoxActionsToReals)?

**Pull request closed without merging**